packages/frontend/src/components/UserSettings/UserAttributesPanel/UserAttributeModal.tsx

```diff
@@ -136,15 +136,20 @@ const UserAttributeModal: FC<{
                     {userAttribute ? 'Update' : 'Add'} user attribute
                 </Title>
             }
+            yOffset={65}
```

Comment:
is the offset from top? we should probably abstract this at some point

Reply:
Yea, it's from the top. For dialogs that can grow it's best not to start them in the middle of the page. Agreed we should abstract it. I think a lot of this dialog style should be centralized. I'm gonna merge this, but I do want to address shared dialog styles soon.